course-definition.yml

```diff
@@ -3878,3 +3890,58 @@ stages:
     name: "Blocking retrieval with timeout"
     difficulty: medium
     marketing_md: In this stage, you will add support for a non-zero timeout duration for the `BLPOP` command.
+
+  # Geospatial Commands
+  - slug: "zt4"
+    primary_extension_slug: "geospatial"
+    name: "Add a location"
```

Comment:
Great work on the names, this is perfect! Only note is that "Validate Coordinates" doesn't need capitalization - just do "Validate coordinates".